lefthook-local.yml.example

```diff
@@ -0,0 +1,7 @@
+# See following doc for details
+# https://github.com/evilmartians/lefthook/blob/master/docs/configuration.md#rc
+
+# You can choose whatever name you want.
```

Comment:
Maybe we should have a message that says something like `we recommend you use 'lefthook-local.yml' as the name to avoid accidentally commiting your lefthook file but feel free to use any filename`

Reply:
`You can choose whatever name you want` means the `~/.lefthookrc` file
I have updated comment to be clearer